components/ActionButton.tsx

```diff
@@ -20,11 +20,11 @@ const ActionButton: FC<ActionButtonProps> = ({
   switch (style) {
     case 'primary':
       classStyle +=
-        'border-blue-dark bg-blue-dark text-basic-white focus:bg-blue-normal hover:bg-blue-normal active:bg-blue-active'
+        'border-blue-dark bg-blue-dark text-basic-white focus:bg-blue-normal hover:bg-blue-normal active:bg-blue-active shadow-sm'
       break
     default:
       classStyle +=
-        'border-gray-default bg-gray-light text-blue-light hover:border-gray-deep hover:bg-gray-dark'
+        'border-gray-dark bg-gray-normal text-blue-light hover:border-gray-deep hover:bg-gray-dark focus:text-blue-light focus:bg-gray-dark shadow-sm'
```

Comment:
Going off the design pattern for the secondary button [here](https://design.canada.ca/common-design-patterns/buttons.html) we can see that the button has an outset border. There isn't an outset border style in tailwind but we can simulate that by setting borders on individual sides like the below suggestion.

```suggestion
        'border-gray-dark bg-gray-normal text-blue-light hover:bg-gray-dark hover:border-l-gray-deep hover:border-t-gray-deep focus:text-blue-light focus:bg-gray-dark shadow-sm border-r-gray-500 border-b-gray-500'
```

Reply:
very nice. I was bummed that there wasn't a way to do it like 'border-r-b...'. good suggestion

Reply:
Tailwind is a great tool that makes styling easy, but it still can't hold a candle to what you can do with conventional CSS

Reply:
omg i agree